CHANGES/1067.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Reverted RFC3986 compatible URL.join honoring empty segments which was introduced in :issue:`1039`.
```

Comment:
@bdraco FYI, Sphinx has a dedicated role `:rfc:` which should be used to link RFCs. The rendered style would also be more consistent.

Reply:
I did discover that, and I changed it before I published it in the release step.  At least I hope I did as my burnout level was pretty high at that point